brax/envs/fast.py

```diff
@@ -19,34 +19,54 @@
 from brax.envs.base import PipelineEnv, State
 import jax
 from jax import numpy as jp
+from flax.core import FrozenDict
 
 
 class Fast(PipelineEnv):
   """Trains an agent to go fast."""
 
-  def __init__(self, **kwargs):
+  def __init__(
+    self,
+    use_dict_obs: bool = False,
+    asymmetric_obs: bool = False,
+    pixel_obs: bool = False,
+    state_obs: bool = True,
```

Comment:
these bools are getting a bit gnarly, can (use_dict_obs, pixel_obs and state_obs) be a single enum? From the user perspective, we could pass a string, and convert to enum as input validation within the class